UnitTests/UnitTests.UWP/UI/Controls/Test_ListDetailsView_UI.cs

```diff
@@ -0,0 +1,113 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Toolkit.Uwp;
+using Microsoft.Toolkit.Uwp.UI;
+using Microsoft.Toolkit.Uwp.UI.Controls;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.ObjectModel;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Markup;
+
+namespace UnitTests.UWP.UI.Controls
+{
+    [TestClass]
+    public class Test_ListDetailsView_UI : VisualUITestBase
+    {
+        private const string SampleXaml = @"<controls:ListDetailsView
+                                                    xmlns=""http://schemas.microsoft.com/winfx/2006/xaml/presentation""
+                                                    xmlns:x=""http://schemas.microsoft.com/winfx/2006/xaml""
+                                                    xmlns:controls=""using:Microsoft.Toolkit.Uwp.UI.Controls""
+                                                    NoSelectionContent=""No item selected"" >
+                                                    <controls:ListDetailsView.ItemTemplate>
+                                                        <DataTemplate>
+                                                            <TextBlock Text=""Item"" />
+                                                        </DataTemplate>
+                                                    </controls:ListDetailsView.ItemTemplate>
+                                                    <controls:ListDetailsView.DetailsTemplate>
+                                                        <DataTemplate>
+                                                            <TextBox Text=""{Binding}"" />
+                                                        </DataTemplate>
+                                                    </controls:ListDetailsView.DetailsTemplate>
+                                                 </controls:ListDetailsView>";
+
+        [TestCategory("ListDetailsView")]
+        [TestMethod]
+        public async Task Test_LoseFocusOnNoSelection()
```

Comment:
When first running both of these unit tests, they both failed.  After running each of them in the Visual Studio debugger, and having them fail, again, they finally _did_ pass, after the next pass.